frontend/src/pages/chat/Chat.tsx

```diff
@@ -530,7 +523,9 @@ const Chat = () => {
     }, [processMessages]);
 
     useEffect(() => {
-        if (AUTH_ENABLED !== undefined) getUserInfoList();
+      if (AUTH_ENABLED) {
```

Comment:
It appears to me that this is a boolean value. From what is see this value is set using this logic in app.py:
`AUTH_ENABLED = os.environ.get("AUTH_ENABLED", "true").lower() == "true"`

Reply:
Just tested the value. Evaluating `AUTH_ENABLED `with `typeof `always results in boolean. I updated my env variable to empty, `Cat` and `undefined `and only received boolean values of false in those instances. Does that work or am I missing part of the question?

Reply:
I believe the original reason for the if check is to determine if that AUTH_ENABLED env variable is set not whether it is true or not

Reply:
If it is set to true, it will continue to this check : 
`if (userInfoList.length === 0 && window.location.hostname !== "127.0.0.1") {
        setShowAuthMessage(true);
 }`

in the original, it starts with a check to see if `AUTH_ENABLED `is false. If so, it sets `showAuthMessage(false)` which is now default. If it makes it past this check, then the value is `true `and does the `userInfoList `and `hostname` check. I believe this is still the same with the current change. 

` 
        if (!AUTH_ENABLED) {
            setShowAuthMessage(false);
            return;
        }
       `

Reply:
Setting `AUTH_ENABLED `to be `undefined `for testing does not result get past the check in useEffect.

Reply:
After further testing on the original version with @tendau, there is a need to show the Auth message UI if `AUTH_ENABLED === undefined` specifically. This will be updated.

Reply:
This has been added to the latest push to this branch.